draft-ietf-jsonpath-base.md

```diff
@@ -1490,18 +1493,21 @@ Notes:
 * `ValueType` is an abstraction of a JSON value or `Nothing`.
 * `LogicalType` is an abstraction of the result of a `logical-expr`.
   Its two instances, `LogicalTrue` and `LogicalFalse`, are not related to
-  the JSON literals `true` and `false` and have no syntactical representation in JSONPath.
+  the JSON literals `true` and `false` and have no direct syntactical representation in JSONPath.
```

Comment:
The "direct" stuff should be done directly to #410

Reply:
Agreed, but it should also be done directly to the main branch, independently of #410. Please see/approve/merge https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/413.